reconcile/mr_client_gateway.py

```diff
@@ -0,0 +1,45 @@
+import reconcile.queries as queries
+
+from utils.sqs_gateway import SQSGateway
+from utils.gitlab_api import GitLabApi
+
+
+class MRClientGatewayError(Exception):
+    """
+    Used when an error happens in the MR Client Gateway initialization.
+    """
+
+
+def init(gitlab_project_id=None, sqs_or_gitlab=None):
+    """
+    Creates the Merge Request client to of a given type.
+
+    :param gitlab_project_id: used when the client type is 'gitlab'
+    :param sqs_or_gitlab: 'gitlab' or 'sqs'
+    :return: an instance of the selected MR client.
+    """
+    if sqs_or_gitlab is None:
+        settings = queries.get_app_interface_settings()
+        client_type = settings.get('pullRequestGateway', 'gitlab')
```

Comment:
change to mergeRequestGateway while we are renaming this component?

Reply:
done